src/Bindings/PluginManager.cpp

```diff
@@ -209,33 +202,45 @@ void cPluginManager::Tick(float a_Dt)
 		ReloadPluginsNow();
 	}
 
-	FIND_HOOK(HOOK_TICK);
-	if (Plugins != m_Hooks.end())
+	auto Plugins = m_Hooks.find(HOOK_TICK);
+	if (Plugins == m_Hooks.end())
 	{
-		for (PluginList::iterator itr = Plugins->second.begin(); itr != Plugins->second.end(); ++itr)
-		{
-			(*itr)->Tick(a_Dt);
-		}
+		return;
+	}
+
+	for (auto * Plugin : Plugins->second)
+	{
+		Plugin->Tick(a_Dt);
 	}
 }
 
 
 
 
 
-bool cPluginManager::CallHookBlockSpread(cWorld & a_World, int a_BlockX, int a_BlockY, int a_BlockZ, eSpreadSource a_Source)
+template <class HookFunction>
```

Comment:
I think I'd prefer `typename` here; although they are both the same, I use `class` when the template param is supposed to be an actual class, and `typename` when it's supposed to be something else, type or function.